Update EGrabber SDK Version

KuktaeKim opened the pull request:
In this pull request, I added the following features:

1. Updated the EGrabber SDK
Upgraded the EGrabber SDK version from egrabber-linux-x86_64-24.01.2.67 to egrabber-linux-x86_64-25.02.0.39.
This update required changing **_EgrabberClientVersion_** to **_EGrabberApiVersion_**.

bhill-slac commented on EuresysApp/src/ADEuresys.cpp:
@KuktaeKim Please rework this pull request as it undoes the support Mark added for SLAC in ADEuresys::createFeature() for the new feature name string prefixes like DV_, IF_, and SY_.
This pull request should also update the egrabber SDK version in iocs/EuresysIOC/configure/CONFIG_SITE.linux-x86_64.Common.
I would also recommend you move the support for NDBitsPerPixel to a separate pull request as Mark hasn't included our NDBitsPerPixel support in ADCore master branch.

KuktaeKim replied:
Hi @bhill-slac,

Thank you for your review. I’ll rework this PR and resubmit it.

KuktaeKim replied:
> @KuktaeKim Please rework this pull request as it undoes the support Mark added for SLAC in ADEuresys::createFeature() for the new feature name string prefixes like DV_, IF_, and SY_. This pull request should also update the egrabber SDK version in iocs/EuresysIOC/configure/CONFIG_SITE.linux-x86_64.Common. I would also recommend you move the support for NDBitsPerPixel to a separate pull request as Mark hasn't included our NDBitsPerPixel support in ADCore master branch.

I’ve reworked this pull request to only update EgrabberClientVersion to EGrabberApiVersion. Updating configure/CONFIG_SITE.linux-x86_64.Common is unnecessary because the README.md already states that "configure/CONFIG_SITE or CONFIG_SITE.$(EPICS_HOST_ARCH).Common must be edited to point to the local install." The user can create a symbolic link to the current SDK version (e.g., egrabber -> /opt/euresys/egrabber-linux-x86_64-25.02.0.39).

KuktaeKim committed:
EgrabberClientVersion changed to EGrabberApiVersion

This change is happend between egrabber-linux-x86_64-24.01.2.67
and egrabber-linux-x86_64-25.02.0.39.
